tests/test_transformations.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+import math
+import random
+import os
+
+import pytest
+import h5py
+import numpy as np
+import cv2
+import imageio
+
+import segmentation_labeling_app.transforms.transformations as transformations
+
+
+@pytest.fixture()
+def video_fixture():
+    current_path = (Path(__file__)).parent
+    video_path = current_path / 'resources' / 'test_video.h5'
+    yield video_path
+
+
+def test_video_random_downsampling(video_fixture):
+    input_fps = 31
+    output_fps = 4
+    new_video = transformations.downsample_h5_video(video_path=video_fixture,
+                                                    input_fps=input_fps,
+                                                    output_fps=output_fps)
+
+    with h5py.File(video_fixture, 'r') as open_video:
+        video = open_video['data'][:]
+        match_count = 0
+        new_video_index = 0
+        sampling_ratio = int(input_fps / output_fps)
+        expected_count = math.ceil(len(video) / int((input_fps / output_fps)))
+
+        for i in range(0, len(video), sampling_ratio):
```

Comment:
This applies to all similar tests -- you should be able to generate exactly the data you expect, and then use numpy.testing module to assert array equality.